src/App/pages/RecipePage/components/RenderPage/RenderPage.module.scss

```
@@ -0,0 +1,91 @@
+$pageWidth: 752px;
+%conteiner {
+  width: $pageWidth;
```

лучше использовать max-width + width
```suggestion
  max-width: $pageWidth;
  width: 100%
```

В таком случае вам контейнер будет растягивать до $pageWidth, но на маленьких экранах он будет уменьшатся